torch/testing/_internal/common_quantization.py

```diff
@@ -408,12 +408,19 @@ def __init__(self):
         self.fc1 = torch.nn.Linear(5, 8).to(dtype=torch.float)
         self.layer_norm = torch.nn.LayerNorm((8))
         self.group_norm = torch.nn.GroupNorm(2, 8)
+        # TODO: add handling for affine=False (future PR)
```

Comment:
For my understanding, why is affine=False different?

Reply:
it's a bug as of this revision, fixed it further upstream in https://github.com/pytorch/pytorch/pull/39203